search_test.go

```diff
@@ -1,12 +1,10 @@
 // Copyright 2020-21 PJ Engineering and Business Solutions Pty. Ltd. All rights reserved.
 
-package googlesearch_test
+package googlesearch
```

Comment:
This needs to be reverted back to blackbox testing mode.

Reply:
Restored in dfdb3593e1e2388020736c362ba2bd21d4b6222e